templates/hash.html

```diff
@@ -1,7 +1,7 @@
 {{! This Source Code Form is subject to the terms of the Mozilla Public
     License, v. 2.0. If a copy of the MPL was not distributed with this
     file, You can obtain one at http://mozilla.org/MPL/2.0/. }}
-<!DOCTYPE HTML>
+<!doctype html>
```

Comment:
Out of curiosity, does this actually matter? Is one more correct than the other?